FlowCryptAppTests/Core/FlowCryptCoreTests.swift

```diff
@@ -93,104 +93,80 @@ class FlowCryptCoreTests: XCTestCase {
         XCTAssertThrowsError(try core.decryptKey(armoredPrv: TestData.k0.prv, passphrase: "wrong"))
     }
 
-    func testComposeEmailPlain() throws {
-        let msg = SendableMsg(text: "this is the message", to: ["email@hello.com"], cc: [], bcc: [], from: "sender@hello.com", subject: "subj", replyToMimeMsg: nil, atts: [], pubKeys: nil)
-        let expectation = XCTestExpectation()
-        
-        var mime: String = ""
-        core.composeEmail(msg: msg, fmt: .plain, pubKeys: nil)
-            .sinkFuture(
-                receiveValue: { composeEmailRes in
-                    mime = String(data: composeEmailRes.mimeEncoded, encoding: .utf8)!
-                    expectation.fulfill()
-                }, receiveError: {_ in }
-            )
-            .store(in: &cancellable)
-        wait(for: [expectation], timeout: 3)
+    func testComposeEmailPlain() async throws {
```

Comment:
Test framework allows to use `async` keyword and simplify testing of async code. No need for expectations.